lib/TPP/GPU/GpuPipeline.cpp

```diff
@@ -127,22 +153,22 @@ struct GpuPipeline : public tpp::impl::GpuPipelineBase<GpuPipeline>,
     pm.addPass(createConvertForAllToParallelOp());
     pm.addNestedPass<func::FuncOp>(createCleanup());
 
+    // Default warp tile sizes.
+    SmallVector<int64_t> warpTileSizes = {16, 16, 16};
```

Comment:
Don't leave default values spread around the code. Either make it default option in the function signature or add a command line option with a default value.

Reply:
Added CLI options as three separate flags as `cl::opt` does not accept SmallVector.

Reply:
Isn't there a version that accepts a string list like "16,16,16", then we split?

Reply:
Found a way to do that through `llvm::cl` 👍